toolchain/binutils.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import build
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+backend = SoftwareManager()
+
+
+class Binutils(Test):
+    """
+    This testcase make use of unit testsuite provided by the
+    source package, performs functional test for all binary tools
+    source file is downloaded and compiled.
+    """
+
+    def setUp(self):
+        '''
+        Install required tools and resolve dependencies
+        '''
+        backend.install('rpmbuild')
+        backend.install('elfutils')
+        backend.install('build')
+        backend.install('autoconf')
+        backend.install('automake')
+        backend.install('binutils-devel')
+        backend.install('dejagnu')
+        backend.install('libtool')
```

Comment:
There are couple of dependencies required on Fedora.23. Could you please add: `glibc-static` and `zlib-static`?